Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/fubar/ubuntu-indexer
+MAINTAINER Peter
+
+# Install Rust.
+RUN curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly --date=2015-06-14 --yes
+# It's okay Mercurial, you can trust staff group.
+RUN echo [trusted] >> /home/jenkins/.hgrc && echo groups = staff >> /home/jenkins/.hgrc
+# Replace the distributed dxr with the one from here (working copy).
+ADD . /builds/dxr-build-env/dxr
+# Run make.
+RUN . venv/bin/activate && \
+    dxr/peep.py install -r dxr/requirements.txt && \
+    cd dxr && \
+    python setup.py install && \
+    make && \
+    cd - && \
+    deactivate
```

Comment:
Hmm, I borrowed this from https://github.com/klibby/dxr-docker/blob/00f7b4fdf86420d9029fdac445c67fb7f98d6ecc/docker/ubuntu-indexer/bin/dxr-setup.sh#L40-L43